aiida/transports/plugins/ssh.py

```diff
@@ -437,14 +437,26 @@ def open(self):
             )
             raise
 
-        # Open also a SFTPClient
-        self._sftp = self._client.open_sftp()
-        # Set the current directory to a explicit path, and not to None
-        self._sftp.chdir(self._sftp.normalize('.'))
+        # Open also a File transport client. SFTP by default, pure SSH in ssh_only
+        self.open_file_transport()
+
+        return self
+
+    def open_file_transport(self):
+        """
+        Open the SFTP channel, and handle error by directing customer to try another transport
+        """
+        from aiida.common.exceptions import InvalidOperation
+
+        try:
+            self._sftp = self._client.open_sftp()
+        except Exception:
+            raise InvalidOperation('SFTP seems not functional. Try using sshonly plugin instead?')
```

Comment:
I would use the full entry point name to make it more clear what `sshonly` actually is, e.g.:
`Try setting up the computer with the `aiida.schedulers:ssh_only` plugin. Note that I would also call it `ssh_only` which is more readable, I think

Reply:
I added the ssh_transports:sshonly bit. The name might be changed soon, I have to check a few things before

Reply:
I changed the port name in the plugin, published it, it's now ssh_transports:ssh_only. This is reflected in 
9f52676